src/components/DateTimePicker/_helpers/custom_locale/hy/_lib/localize/index.js

```diff
@@ -0,0 +1,180 @@
+import buildLocalizeFn from '../../../../../../../../node_modules/date-fns/locale/_lib/buildLocalizeFn'
+
+
+var eraValues = {
+  narrow: ['Ք', 'Մ'],
+  abbreviated: ['ՔԱ', 'ՄԹ'],
+  wide: ['Քրիստոսից առաջ', 'Մեր թվարկության']
```

Comment:
This is the translation to "Anno Domini", as in opposite to Մեր թվարկությունից առաջ։ Could have used Քրիստոսից հետո, but then the first letter would be the same in case of "narrow" representation